test/models/term_test.rb

```diff
@@ -448,4 +449,18 @@ class TermTest < ActiveSupport::TestCase
     # Verify impact
     assert_nil target_term.cluster
   end
+
+  test 'does not destroy records with suggested resource' do
+    term = terms(:jstor)
+    assert term.suggested_resource
+
+    term.destroy
+    assert term.present?
+  end
+
+  test 'destroys record successfully if no suggested resource is present' do
+    term = terms(:hi)
+    assert_nil term.suggested_resource
+    assert_nothing_raised { term.destroy }
```

Comment:
Nothing is raised in the other test so Im not sure this is testing what we think it is? Maybe a better assertion is to confirm `Term.where(phrase: 'whatever the original term we just deleted phrase was')` returns no match?

Reply:
Or maybe just follow what the other test does and check for `term.present?` If that doesn't work as expected then both tests would need updating :)

Reply:
Oh yeah, good catch. I think this was from an earlier iteration where I was raising an exception rather than aborting the callback. I agree that `assert_not term.present?` is much better here.